packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/src/env.d.ts

```diff
@@ -0,0 +1,10 @@
+declare global {
+    namespace NodeJS {
+      interface ProcessEnv {
+        locales: string[];
+        DEBUG_MODE: boolean;
+      }
+    }
+  }
+
+export {}
```

Comment:
```suggestion
export declare global {
	namespace NodeJS {
		interface ProcessEnv {
			locales: string[];
			DEBUG_MODE: boolean;
		}
	}
}

```

remove empty export, add newline, use tabs > spaces

Reply:
Since this file does not have any import/export statements the empty export is used to convert this to a module and is needed for the types to get recognized in this format
https://stackoverflow.com/questions/45194598/using-process-env-in-typescript

Reply:
does `export declare global` not work?

Reply:
Just adjusted to that and pushed the change 👍